tests/framework/utils_iperf.py

```diff
@@ -152,8 +152,11 @@ def guest_command(self, port_offset):
 
 def emit_iperf3_metrics(metrics, iperf_result, omit):
     """Consume the iperf3 data produced by the tcp/vsock throughput performance tests"""
-    for cpu_util in iperf_result["cpu_load_raw"]["firecracker"]:
-        metrics.put_metric("cpu_utilization_vmm", cpu_util, "Percent")
+    cpu_util = iperf_result["cpu_load_raw"]
+    for thread_name in cpu_util:
+        values = cpu_util[thread_name]
+        for value in values:
+            metrics.put_metric(f"cpu_utilization_{thread_name}", value, "Percent")
```

Comment:
I run A/B tests several times and the block and net tests pass. I think the cpu usage should be not change much, because we usually have number of tasks equal to number of vcpus.

Reply:
alright, thanks for checking!